tests/framework/installer/install_helper.go

```diff
@@ -222,61 +217,62 @@ func (h *InstallHelper) UninstallRookFromK8s(clusterNamespace string, helmInstal
 	logger.Infof("Uninstalling Rook")
 	k8sHelp, err := utils.CreateK8sHelper(h.T)
 	if err != nil {
-		panic(err)
+		logger.Errorf("cannot uninstall rook err ->  %v", err)
```

Comment:
in this method, should we use assert.Nil(t, err) instead of just logging the error. We will likely not notice the error since it doesn't block the test. Or maybe we don't want to fail the test?

Reply:
i think this PR has this problem in general in other places.  In a lot of places, we've replaced a panic with just a log statement, when we probably want to return the failure to the caller or assert/require to fail the test.

If we simply log an error, we'll run into problems later on that may not be immediately obvious that they are the result of a previous failure.  That will make debugging more difficult.

Reply:
the install section is returning erros now(instead of panic) and the tests assert those errors. However i have added asserts to the uninstall func instead of returning errors.